### PR TITLE
DSERV-198 updating db collection names

### PR DESCRIPTION
**Pull request by pedrohr**

### CODE REVIEW

File: data/schema-config.yaml [hunk 1]
```diff
@@ -497,7 +497,7 @@ gtex splice variant to gene association:
   represented_as: edge
   label_in_input: GTEx_splice_QTL
   label_as_edge: GTEX_SPLICE_QTL
-  db_collection_name: variant_gene_links
+  db_collection_name: variant_genes
```
Comment (mingjiecn) [inline data/schema-config.yaml:500]: should be variants_genes?

File: data/schema-config.yaml [hunk 2]
```diff
@@ -535,7 +535,7 @@ gtex variant to gene expression association:
   represented_as: edge
   label_in_input: GTEx_eqtl
   label_as_edge: GTEx_eQTL
-  db_collection_name: variant_gene_links
+  db_collection_name: variant_genes
```
Comment (mingjiecn) [inline data/schema-config.yaml:538]: should be variants_genes?

**Review by mingjiecn — CHANGES_REQUESTED** [review]
See my comment for requested changes. Thank you!

File: data/schema-config.yaml (hunk 1 above)
Comment (pedrohr) [reply data/schema-config.yaml]: yes, good catch! just pushed the change

**Review by mingjiecn — APPROVED** [review]

**Review by mingjiecn — APPROVED** [review]

**Pull request merged**